docs/configuration.md

```diff
@@ -2018,6 +2018,50 @@ Apart from these, the following properties are also available, and may be useful
 </tr>
 </table>
 
+### Barrier Mode Execution
+
```

Comment:
I put this section below the scheduling section because this is related to scheduling.